SwiftUI_DZ/SwiftUI_DZ/View/VPNView.swift

```diff
@@ -0,0 +1,218 @@
+        static let mbText = "MB"
+        static let pleaseConnectText = "Please Connect"
+        static let countryText = "Zimbabwe"
+        static let expireText = "Бесплатная версия сервиса доступна в течении 15 дней"
+        static let ipAddressValueText = "127.0.0.345"
+        static let zeroIntValue = 0
+        static let countryImageName = "Zimbabwe"
+        static let vstackSpacing: CGFloat = 20
+        static let startRadius: CGFloat = 0
+        static let endRadius: CGFloat = 100
+        static let countryImageFrame: CGFloat = 150
+        static let countryImageCornerRadius: CGFloat = 75
+        static let countryStrokeLineWidth: CGFloat = 3
+        static let countryShadowRadius: CGFloat = 10
+        static let countryBlurRadius: CGFloat = 10
+        static let cgFloatZero: CGFloat = 0
+        static let onOfTextCornerRadius: CGFloat = 15
+        static let onOfTextStrokeLineWidth: CGFloat = 3
+        static let onOfShadowRadius: CGFloat = 4
+        static let timeTextCircleFrame: CGFloat = 100
+        static let timeCircleFrame: CGFloat = 180
+        static let timeCircleTrimFrom: CGFloat = 0
+        static let timeCircleTrimTo: CGFloat = 1
+        static let timeCircleRotationEffect: CGFloat = 90
+        static let oneDoubleCount = 1.0
+        static let strokelineWidth: CGFloat = 15
+    }
+    
+    // MARK: - Public Properties
+    var body: some View {
+        NavigationView {
+            ScrollView {
+                ZStack {
+                    Color.black
+                    VStack(spacing: Constants.vstackSpacing) {
+                        switch isToggleOn {
+                        case true:
+                            ZStack {
+                                ZStack {
+                                    CountryFlagImageView
+                                    timeTextView
+                                }
+                                timeCircle
+                            }
+                            countryText
+                            onOffTextView
+                        case false:
+                            CountryFlagImageView
+                            countryText
+                            onOffTextView
+                        }
+                        
+                        onToggleView
+                        VStack {
+                            ipAddressTextView
+                            receivedTextView
+                            sentTextView
+                        }
+                        .padding()
+                        Spacer()
+                        expireTextView
+                    }
+                    .animation(.default)
+                }
+            }
+        }
+    }
+    
+    // MARK: - Private Properties
+    @StateObject private var stopWatchManager = StopWatchManager()
+    @State private var isToggleOn = false
+    @State private var onoff = false
+    private let smallRadial = RadialGradient(
```

Comment:
```suggestion
    @State private var onoff = false
    
    private let smallRadial = RadialGradient(
```
разделяй разнотипные проверти для улучшения читаемости кода)